src/components/Button/__tests__/Button.spec.js

```diff
@@ -35,4 +36,10 @@ describe('<Button> that renders a button', () => {
         buttonWrapper.simulate('mouseover');
         expect(onMouseOverMock).toHaveBeenCalled();
     });
+    it('should render an ancor element if href is defined', () => {
+        const wrapper = mount(<Button href="/">Click me</Button>);
+        expect(toJson(wrapper)).toMatchSnapshot();
+        expect(wrapper.find('button')).toHaveLength(0);
```

Comment:
Is this assertion necessary? If it finds an `a`, doesn't that implicitly mean it does not find a `button`?